pkg/cluster/ipaddresses.go

```diff
@@ -161,6 +163,9 @@ func (m *manager) populateDatabaseIntIP(ctx context.Context) error {
 // refers to -pip-v4, which doesn't exist on pre-DNS change clusters.
 func (m *manager) updateAPIIPEarly(ctx context.Context) error {
 	infraID := m.doc.OpenShiftCluster.Properties.InfraID
+	if infraID == "" {
```

Comment:
These aren't actually needed because `updateAPIIPEarly` and `createAPIServerPrivateEndpoint` functions are only run on install which happens after our `ensureInfraId` function.